### PR TITLE
Added CI

### PR DESCRIPTION
**Pull request by windymelt**
temurinでJava 8, 11, 17でsbt testを走らせています

### CODE REVIEW

**Review by tarao — COMMENTED** [review]
Maybe we should put a badge on README?

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: CI
+on:
+  pull_request:
+  push:
```
Comment (tarao) [inline .github/workflows/ci.yaml:4]: This triggers CI twice for pull requests.  It should be the following I guess.

```
  push:
    branches:
    - master
```

**Review by tarao — CHANGES_REQUESTED** [review]

**Comment by windymelt** [comment]
Added CI badge on header of `README.md`. (Where to place may be controversial)

Prevented duplicated CI trigger.

**Review by tarao — APPROVED** [review]
@windymelt LGTM

**Pull request merged**